libs/common/src/types/mongo-db-models/facility.ts

```diff
@@ -0,0 +1,44 @@
+import { ObjectId } from 'mongodb';
+import { UnixTimestamp } from '../date';
+import { Currency } from '../currency';
+
+/**
+ * Type of the mongo db "facilities" collection
```

Comment:
I've made the decision to remove the parsing methods from the tfm facilities repo so that we defer to this type as the default from now on and don't have two sources of truth to maintain